src/tree/containers/ContainerTreeItem.ts

```diff
@@ -169,7 +169,7 @@ const containerTooltipTemplate = `
 ---
 
 #### Image
-{{ imageName }} ({{ substr imageId 7 12 }})
+{{ image.originalName }} ({{ substr imageId 7 12 }})
```

Comment:
Not related to this bug/PR, but this and the line below are some super easy fixes for some tooltip issues in the Containers view. Figured it was easier to just fix them than to create an issue, triage, etc. etc.